src/proxy-ui-api/frontend/src/views/Clients/ClientsDataTable.vue

```diff
@@ -113,17 +122,25 @@ import Vue from 'vue';
 import ClientStatus from './ClientStatus.vue';
 import { mapGetters } from 'vuex';
 import { Permissions, RouteName } from '@/global';
+import { Client } from '@/types';
+import SmallButton from '@/components/ui/SmallButton.vue';
+import ConfirmDialog from '@/components/ui/ConfirmDialog.vue';
 
 export default Vue.extend({
   components: {
     ClientStatus,
+    SmallButton,
+    ConfirmDialog,
   },
 
   data: () => ({
     search: '',
     pagination: {
       sortBy: 'sortNameAsc',
     },
+    confirmRegisterClient: false,
```

Comment:
add typings. And while you're on to it add typings to all variables in the data-block

Reply:
Fixed